app/controllers/api/v1/BookmarkletController.js

```diff
@@ -48,7 +48,12 @@ const getAttachments = async function(author, imageUrl) {
   const newAttachment = await author.newAttachment({ file })
   await newAttachment.create()
 
-  return [newAttachment.id]
+  return newAttachment.id
+}
+
+const getAttachments = async function(author, imageUrls) {
+  const promises = imageUrls.map(async (url) => await getAttachment(author, url))
```

Comment:
no need for `await` inside of arrow.
no need to mark arrow as `async` even, as it just forwards the promise as a value further on